### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by brucify**
Updating the Github README.md page with simplified content, examples and instruction to get help and get started.

### CODE REVIEW

**Comment by essen** [comment]
The example is not productive because it does not say how to run it. Existing users don't need it, and newcomers will only see the "ugly" syntax with no ways to even try it.

**Comment by brucify** [comment]
Thanks @essen for the suggestion. Updated the example with how to run. Open to suggestions.

File: README.md [hunk 1]
```diff
@@ -1,60 +1,79 @@
...
-Building and Installing
------------------------
+Eshell V8.2  (abort with ^G)
+1> c(hello).
+{ok,hello}
+2> hello:hello_world().
+Hello, world
+ok
+```
+This example defines a function `hello_world/0` that prints "Hello, world" in the Erlang shell.
```
Comment (essen) [inline README.md]: You can probably move sentence right before showing it. Otherwise, :+1:

Comment (brucify) [inline README.md]: Updated. Thanks ✌️

**Comment by crownedgrouse** [comment]
hello:world()  instead would be better Mho ?

**Pull request merged**